packages/procps/Makefile.in.patch

```diff
@@ -0,0 +1,12 @@
+--- procps-3.3.17.orig/Makefile.in	2021-02-09 17:11:25.000000000 +0700
++++ procps-3.3.17/Makefile.in	2021-05-11 23:54:58.000000000 +0700
+@@ -139,7 +139,7 @@
+ @WITH_NCURSES_TRUE@	watch.1 \
+ @WITH_NCURSES_TRUE@	top/top.1
+ 
+-@CYGWIN_TRUE@@WITH_NCURSES_TRUE@am__append_20 = lib/strverscmp.c
++@WITH_NCURSES_TRUE@am__append_20 = lib/strverscmp.c
```

Comment:
build.sh executes `autoreconf -fi` before configure script is being run. Therefore it is necessary only to patch *.am files.